pkg/skuba/actions/node/upgrade/apply.go

```diff
@@ -75,12 +75,34 @@ func Apply(client clientset.Interface, target *deployments.Target) error {
 		return err
 	}
 
+	// Disable skuba-update.timer before upgrade
+	if skubaUpdateWasEnabled {
+		err = target.Apply(nil, "skuba-update-timer.disable")
+		if err != nil {
+			return err
+		}
+	}
+
+  // Check if a kured reboot file already exists
+  if kured.RebootFileExists() {
```

Comment:
indentation seems a little off, compared to the next paragraph L93+.

Reply:
you should also record `kuredRebootFilePresent` here, for line 216.

Reply:
done